src/AzSK.ADO/Framework/Configurations/SVT/ADO/ADO.CommonSVTControls.json

```diff
@@ -95,6 +95,23 @@
         ],
         "Enabled": true
     },
+    {
+      "ControlID": "ADO_Feed_DP_Review_Inactive_Feeds",
+      "Description": "Inactive feeds must be removed if no more required.",
+      "Id": "Feed120",
+      "ControlSeverity": "High",
+      "Automated": "Yes",
+      "MethodName": "CheckForInactiveFeeds",
+      "Rationale": "Each additional feed increases the attack surface. To minimize this risk ensure that only active and legitimate feeds are present in your environment.",
```

Comment:
Rationale doesn't specify how the attack surface is increased, may need to review it from Manish.